database/src/main/kotlin/no/nav/su/se/bakover/database/revurdering/RevurderingPostgresRepo.kt

```diff
@@ -138,30 +141,34 @@ internal class RevurderingPostgresRepo(
                 tilRevurdering = tilRevurdering,
                 beregning = beregning!!,
                 simulering = simulering!!,
-                saksbehandler = Saksbehandler(saksbehandler)
+                saksbehandler = Saksbehandler(saksbehandler),
+                oppgaveId = OppgaveId(oppgaveId!!)
             )
             RevurderingsType.BEREGNET_INNVILGET -> BeregnetRevurdering.Innvilget(
                 id = id,
                 periode = periode,
                 opprettet = opprettet,
                 tilRevurdering = tilRevurdering,
                 beregning = beregning!!,
-                saksbehandler = Saksbehandler(saksbehandler)
+                saksbehandler = Saksbehandler(saksbehandler),
+                oppgaveId = OppgaveId(oppgaveId!!)
             )
             RevurderingsType.BEREGNET_AVSLAG -> BeregnetRevurdering.Avslag(
                 id = id,
                 periode = periode,
                 opprettet = opprettet,
                 tilRevurdering = tilRevurdering,
                 beregning = beregning!!,
-                saksbehandler = Saksbehandler(saksbehandler)
+                saksbehandler = Saksbehandler(saksbehandler),
+                oppgaveId = OppgaveId(oppgaveId!!)
             )
             RevurderingsType.OPPRETTET -> OpprettetRevurdering(
                 id = id,
                 periode = periode,
                 opprettet = opprettet,
                 tilRevurdering = tilRevurdering,
-                saksbehandler = Saksbehandler(saksbehandler)
+                saksbehandler = Saksbehandler(saksbehandler),
+                oppgaveId = OppgaveId(oppgaveId!!)
```

Comment:
Hvordan fungerer dette? Vi lager en oppgave så snart vi har opprettet en ny revurdering? Hva er spesifikasjonen fra fag her?

Reply:
Tanken är att en revurdering skal opprettes når f. eks en melding av endring har kommit. Trodde att de kom in som oppgaver? 😊

Reply:
Alternativet er att bara fjerne att revurdering er knyttet till en oppgave

Reply:
Kan få en avklaring fra Tor Erik her. Hvis disse "melding om endring" allerede ligger som oppgaver, vil de bli "dobbelt opp". Samtidig hvis de er opprettet utenfor systemet, er det ikke så lett å knytte revurderingen til den oppgaven. Da må vi legge inn ekstra støtte for det.